modules/deployment/src/main/java/org/eclipse/xpanse/modules/deployment/ServiceStateManager.java

```diff
@@ -98,7 +100,9 @@ private void asyncStartService(ServiceOrderEntity serviceOrderTaskEntity,
         try {
             result = plugin.startService(request);
         } catch (Exception e) {
-            serviceOrderTaskEntity.setErrorMsg(e.getMessage());
+            serviceOrderTaskEntity.setError(
```

Comment:
Its better we use the methods in SerivceOrderManager class to update in-progress, completed, etc.